src/main.js

```diff
@@ -44,9 +45,71 @@ function ymlContentToConfig(ymlContent) {
   }
 
   if (configJson.resource) {
-    config.source = configJson.resource;
+    config.source = removeTrailingSlashes(configJson.resource);
+    config.multiple = false;
+    config.cacheComparator = compareArrays;
+    config.diffChecker = onlyInLeft;
+    config.resourceUpdater = (del, add) => updateResource(del, add, config.source);
+  } else if (configJson.resources) {
+    config.multiple = true;
+    config.resourceHostmap = configJson.resources.map((object) => {
+      return {
+        resource: removeTrailingSlashes(object.resource),
+        host: removeTrailingSlashes(object.host)
```

Comment:
It might be that slashes can be safely remove here. But not a 100% sure. I would give that a try with CSS to see what happens.

Reply:
The CSS has no issue when there are 2 slashes instead of 1. 
`http://localhost:3000//.well-known/solid` gives the same response as `http://localhost:3000/.well-known/solid`.
But I've read online that it can impact caching.

Reply:
Yeah, for these things I would just leave it to the user for now. They have to provide good input and not then we show the appropriate error.

Reply:
OK, I'll remove it 👍